src/olympia/reviews/tests/test_views.py

```diff
@@ -880,6 +1018,25 @@ def test_edit_owner_admin(self):
         assert response.data['title'] == unicode(self.review.title) == u'Titlé'
         assert response.data['version'] == self.review.version.version
 
+    def test_edit_reply(self):
+        addon_author = user_factory()
+        self.addon.addonuser_set.create(user=addon_author)
+        self.client.login_api(addon_author)
+        reply = Review.objects.create(
+            reply_to=self.review, body=u'This is â reply', user=addon_author,
+            addon=self.addon)
+        self.url = reverse(
+            'addon-review-detail',
+            kwargs={'addon_pk': self.addon.pk, 'pk': reply.pk})
+
+        response = self.client.patch(self.url, {'rating': 5})
+        assert response.status_code == 200, response.content
```

Comment:
we're usually not logging response.content in other asserts, is this still from debugging things?